checks/remotesettings/changes_timestamps.py

```diff
@@ -30,10 +31,21 @@ def get_timestamp(client, bucket, collection, timestamp):
         for entry in entries
     ]
     results = await asyncio.gather(*futures)
-    failing = []
+    all_good = True
+    datetimes = []
     for (entry, timestamp) in zip(entries, results):
+        collection_timestamp = timestamp
+        entry_timestamp = str(entry["last_modified"])
```

Comment:
Why should we convert these to strings? The fact that we're parsing them as millis-since-the-epoch means that they essentially "have" to be integers..

Reply:
Good point, thanks!